node/network/availability-distribution/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "polkadot-availability-distribution"
+version = "0.1.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
+
+[dependencies]
+futures = "0.3.5"
+futures-timer = "3.0.2"
```

Comment:
this one is what you're using `smol::Timer` for? :)
I guess it could be moved to dev-dependencies and replace `smol` there